### PR TITLE
Rake tasks

### PR DESCRIPTION
**Pull request by dkulback**
Invoice Task

### CODE REVIEW

File: spec/models/invoice_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Invoice, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (hannahkwarren) [inline spec/models/invoice_spec.rb]: Should we put a relationships test in here for the customer:invoice relationship? e.g. should belong_to ... and conversely add one to the customer model spec?

**Review by hannahkwarren — COMMENTED** [review]
I left one comment - we can either change it before you merge this, or make a second PR with relationship tests for customer and invoice, your call!

File: spec/models/invoice_spec.rb (hunk 1 above)
Comment (dkulback) [reply spec/models/invoice_spec.rb]: Updated the spec file to have that relationship test

**Comment by hannahkwarren** [comment]
Looks good - just noting that we should add a relationship test on customer as well.

**Pull request merged**